src/test/debuginfo/collapse-debuginfo-with-attr-flag.rs

```diff
@@ -0,0 +1,63 @@
+// ignore-lldb
+#![feature(collapse_debuginfo)]
+
+// Test that line numbers are not replaced with those of the outermost expansion site when the
+// `collapse_debuginfo` is active and `-Zdebug-macros` is provided, despite `#[collapse_debuginfo]`
+// being used.
```

Comment:
This is exactly the same behavior as `src/test/debuginfo/collapse-debuginfo-no-attr.rs`, right? Maybe use test revisions, so the file is not duplicated.

Reply:
I think it's clearer if the tests are kept separate, while the behaviour is the same, the code itself isn't - this test uses the `#[collapse_debuginfo]` attribute while the `no-attr` test does not.

Reply:
OK, that works for me too.